articles/2016-09-01-semantic-html-tags.md

```diff
@@ -0,0 +1,98 @@
+
+It should be stated that this approach only works in certain cases, usually smaller or simpler UI's. Styling on tag selectors is considered a bad practice if the UI is too large to be appropriately represented as such. There are situations where it can work, but you are more likely going to need to use class selectors for your styling.  
+
+### Frameworks and Templating
+
+If you're using a front-end framework like Ember, you will notice that a lot of components will be wrapped in `<div>`s. This is ok, as the benefit of a `<div>` is that it's generic enough to be used everywhere. If you are committed to using semantic tags, you have two choices here. You can either wrap the template inside the component with the appropriate tag,
+
+```handlebars
+{{#some-component}}
+  <section>
+    <!-- section content goes here -->
+  </section>
+{{/some-component}}
+```
+
+This will render to the following HTML
+
+```html
+<div id="ember123">
+  <section>
+    <!-- section content goes here -->
+  </section>
+</div>
+```
+
+Another option (one that I personally prefer), is to change the `tagName` attribute of the component to match the appropriate semantic tag,
+
+```handlebars
+{{#some-component
+  tagName='section'
+}}
+  <!-- section content goes here -->
+{{/some-component}}
+```
+
+Which will render to
+
+```html
+<section id="ember123">
+  <!-- section content goes here -->
+</section>
+```
+
+The reason I prefer this option is because the wrapper `<div>` in choice 1 isn't actually necessary, and it can get in the way of some of the interesting things you can do with relative pseudo-selectors (for a refresher on what that is, here's the [blog post](2016-05-23-selectors-part-2-pseudo-classes.md) I wrote about it).
+
+## Example
+
+Here is an example of what the outline of a page would look like using semantic HTML5 elements.
+
+```html
+<header></header>
+<nav></nav>
+<main>
+  <article>
+    <section></section>
+    <section></section>
+  </article>
+  <aside>
+    <section></section>
+    <section></section>
+    <section></section>
+  </aside>
+</main>
+<footer></footer>
+```
+
+Even without adding any CSS or seeing the content itself, you can easily visualize what the page will look like and how its structured.
+
+For an example of how this might look in a page with content and CSS, check out this [codepen demo](http://codepen.io/raemadeline/pen/bwdgvJ)
+
+## Conclusion
+
+When in doubt, use the HTML element that most directly aligns with the purpose of the content within. `<div>`s have their time and place, but if it's at all possible to replace a `<div>` with a more descriptive element, please do so.
```

Comment:
I think it'd be good to have a "When to use `div`s" section that describes cases where there might not be an appropriate semantic tag.